<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TicketDetails</class>
 <widget class="QWidget" name="TicketDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>788</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="verticalSpacing">
       <number>4</number>
      </property>
      <item row="19" column="2">
       <widget class="QPushButton" name="changeState">
        <property name="text">
         <string>Schließen</string>
        </property>
       </widget>
      </item>
      <item row="15" column="2">
       <widget class="QLabel" name="processedByLabel2">
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item row="16" column="1">
       <widget class="QLabel" name="testedByLabel1">
        <property name="text">
         <string>Getestet von</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QComboBox" name="priorityBox"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="ticketIDLabel">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>TicketID</string>
        </property>
       </widget>
      </item>
      <item row="16" column="2">
       <widget class="QLabel" name="testedByLabel2">
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item row="14" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Erstellt am</string>
        </property>
       </widget>
      </item>
      <item row="18" column="1" colspan="2">
       <widget class="QTextEdit" name="title">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="15" column="1">
       <widget class="QLabel" name="processedByLabel1">
        <property name="text">
         <string>Erledigt von</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Priorität</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Kunde</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="customerLabel">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Erledigt</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Kategorie</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QComboBox" name="categorie"/>
      </item>
      <item row="9" column="2">
       <widget class="QSpinBox" name="percent">
        <property name="suffix">
         <string> %</string>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="14" column="2">
       <widget class="QLabel" name="createdOn">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Deadline</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Abteilung</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QComboBox" name="department"/>
      </item>
      <item row="3" column="2">
       <widget class="QDateTimeEdit" name="deadline">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QLabel" name="createdByLabel1">
        <property name="text">
         <string>Erstellt von</string>
        </property>
       </widget>
      </item>
      <item row="13" column="2">
       <widget class="QLabel" name="createdByLabel2">
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QComboBox" name="ticketToUserBox"/>
      </item>
      <item row="8" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Zugewiesen an</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1" colspan="2">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="19" column="1">
       <widget class="QPushButton" name="reopen">
        <property name="text">
         <string>Wieder öffnen</string>
        </property>
       </widget>
      </item>
      <item row="10" column="2">
       <widget class="QPushButton" name="save">
        <property name="text">
         <string>speichern</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>System</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QComboBox" name="systemBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="history">
     <property name="tabChangesFocus">
      <bool>false</bool>
     </property>
     <property name="documentTitle">
      <string>Verlauf</string>
     </property>
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="tabStopWidth">
      <number>80</number>
     </property>
     <property name="backgroundVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="comment">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="sendToLabel">
       <property name="text">
        <string>Senden an</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="commentToUserBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="send">
       <property name="text">
        <string>Kommentar senden</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
